### PR TITLE
Update test.c

### PR DESCRIPTION
**Pull request by silencio94**
1106

### CODE REVIEW

File: test.c [hunk 1]
```diff
@@ -3,3 +3,4 @@
 
 1111112
 3333333
+4444444s
```
Comment (SophieYoonseo) [inline test.c]: gogogood

**Review by SophieYoonseo — CHANGES_REQUESTED** [review]
request changes

**Review by silencio94 — COMMENTED** [review]
고쳤어요~

**Review by SophieYoonseo — APPROVED** [review]
goodgood

**Pull request merged**